src/pudl/transform/eia860.py

```diff
@@ -1021,3 +1027,124 @@ def _core_eia860__boiler_stack_flue(
     )
 
     return bsf_assn
+
+
+@asset(io_manager_key="pudl_io_manager")
+def _core_eia860__cooling_equipment(
+    raw_eia860__cooling_equipment: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the EIA 860 cooling equipment table.
+
+    - spot clean year values before converting to dates
+    - standardize water rate units to gallons per minute (2009-2013 used cubic
+      feet per second)
+    - convert kilodollars to normal dollars
+
+    Note that the "power_requirement_mw" field is erroneously reported as
+    "_kwh" in the raw data for 2009, 2010, and 2011, even though the values are
+    in MW. This is corroborated by the values for these plants matching up with
+    the stated MW values in later years. Additionally, the PDF form for those
+    years indicates that the value should be in MW, and KWh isn't even a power
+    measurement.
+
+    In 2009, we have two incorrectly entered ``cooling_type`` values of ``HR``,
+    for utility ID 14328, plant IDs 56532/56476, and cooling ID ACC1. This
+    corresponds to the Colusa and Gateway generating stations run by PG&E. In
+    all later years, these cooling facilities are marked as ``DC``, or "dry
+    cooling"; however, ``HR`` follows the coding pattern for hybrid systems. As
```

Comment:
Can you clarify what this means, HR follows the coding pattern for hybrid systems?